linked_list/implementation.py

```diff
@@ -7,34 +8,122 @@ class LinkedList(AbstractLinkedList):
     """
 
     def __init__(self, elements=None):
-        pass
-
+        self.start = None # initialize starting node
```

Comment:
This seems to be a little over documented 😜
No need to explain what you are doing line by line.